deploy/README.md

```diff
@@ -0,0 +1,21 @@
+# Deployment scripts
+## Configure environment
+Configures AWS and installs required software.
+```shell script
+./setup.sh
+```
+_Requires Homebrew and Pip. This script is idempotent_
+
+## Deploy web server
+Use terraform to create and provision an EC2 instance to run the imdown backend.
```

Comment:
this comment is same as for `Create web server`. could you change the comment to reflect how it differs from creating web server?